spec/features/users_can_access_a_page_and_download_a_urn_list.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.feature 'User can access a page' do
+  scenario 'with instructions and a link to the URN list.' do
+    mock_sso_with(email: 'email@example.com')
+    mock_tasks_endpoint!
+    mock_task_with_framework_endpoint!
+
+    visit '/'
+    click_link 'sign-in'
+
+    visit '/tools'
+    expect(page).to have_link('Download CCS URN List (July 2018).xls', href: '/urn/CCS URN List (July 2018).xls')
+
+    expect(page).to have_content('Tools')
```

Comment:
I think the expectation on the link above is enough for this spec. it will give us the confidence that the page renders and that the user has access to the URN file to download. The rest of the expectations do not do much to increase that confidence, and have the downside of making the spec brittle to changes as it forces us to update the spec if we want to make changes to the copy on the page.